src/main/java/com/intentsg/shop/services/impl/CatalogServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.intentsg.shop.services.impl;
+
+import com.intentsg.shop.model.Catalog;
+import com.intentsg.shop.repository.CatalogRepository;
+import com.intentsg.shop.services.CatalogService;
+import org.springframework.context.annotation.Scope;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.stereotype.Service;
+import org.springframework.web.context.WebApplicationContext;
+
+@Scope( value = WebApplicationContext.SCOPE_SESSION, proxyMode = ScopedProxyMode.INTERFACES )
```

Comment:
No reason to make its scope as the session

Reply:
Removed